src/csharp/Grpc.nuspec

```diff
@@ -5,7 +5,7 @@
     <title>gRPC C#</title>
     <summary>C# implementation of gRPC - an RPC library and framework</summary>
     <description>C# implementation of gRPC - an RPC library and framework. See project site for more info.</description>
-    <version>0.5.0</version>
+    <version>0.5.0.1</version>
```

Comment:
Most of the other packages are at 0.9.0, and the C core is already at 0.9.1. We're already out of sync. But my primary concern here is that this would introduce inconsistent version formats across versions, which seems undesirable.